src/components/surfitProfile/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import logo from 'assets/icon_logo.svg';
+import profilePic from 'assets/mockup_profile.svg';
+import { React } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+const NavContainer = styled.ul`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  list-style: none;
+  width: 34.3rem;
+  height: 3.2rem;
+  font-size: 1.4rem;
+  color: ${theme.colors.surfit_grey_3};
+  margin: 5.2rem 0 auto;
+`;
+
+const Img = styled.img`
+  width: 3.2rem;
+  height: 3.2rem;
+`;
+
+const Logo = styled(Img)`
+  border: 1.5px solid white;
+  border-radius: 10rem;
+  rotate: 0.04deg;
+  margin-right: 0.9rem;
+`;
+const Profile = styled(Img)`
+  margin-left: 1.85rem;
+`;
+
+const SelectedCategory = styled.text`
+  font-weight: bold;
```

Comment:
```suggestion
  font-weight: 600;
```
이거 bold 말고 디잔쌤들이 지정해준 weight 쓰는게 좋을듯!